tests/app/Filament/AccessCheckingHelpers/ChecksListingFilamentAccess.php

```diff
@@ -46,4 +47,12 @@ private function indexRoleProvider(): array
      * The text we expect to see on successful index load.
      */
     protected abstract function getIndexText(): array;
+
+    /**
+     * Can be overridden to provide a test fixture for the test if needed.
+     */
+    protected function beforeListing(): void

```

Comment:
Intentionally empty?

Reply:
Yeah - empty default method for the trait :)